cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -641,6 +641,7 @@ protected ConfigMap createConfigMap(String name, Map<String, String> data) {
     protected Volume createSecretVolume(String name, String secretName) {
 
         SecretVolumeSource secretVolumeSource = new SecretVolumeSourceBuilder()
+                //.withDefaultMode(0440)
```

Comment:
Why is the default mode commented out?

Reply:
Sorry I was just trying whether this change has an influence on failing test (I was unable to run it locally).